monitoring/db_update_sqlite.py

```diff
@@ -245,41 +245,52 @@ def refresh_BenchmarksBySubmitHost_from_view(view_name):
     try:
         if view_name == 'VSummaries':
             sql_query = f"""
-            SELECT
-                Site,
-                SubmitHost,
-                ServiceLevelType,
-                ServiceLevel,
-                max(UpdateTime) AS LatestPublish
-            FROM {view_name}
-            WHERE UpdateTime > DATE_SUB(NOW(), INTERVAL 3 MONTH)
-            GROUP BY Site, SubmitHost;
+            SELECT DISTINCT v.Site, v.SubmitHost, v.ServiceLevelType, v.ServiceLevel, v.UpdateTime AS LatestPublish
+            FROM {view_name} v
+            JOIN (
+                SELECT Site, SubmitHost, MAX(UpdateTime) AS LatestPublish
+                FROM {view_name}
+                WHERE UpdateTime > DATE_SUB(NOW(), INTERVAL 3 MONTH)
+                GROUP BY Site, SubmitHost, ServiceLevelType, ServiceLevel
+            ) latest
```

Comment:
For stored SQL, I generally prefer to have explicit aliasing with "AS" (for the table names).